app/assets/javascripts/channels/groups.js

```diff
@@ -0,0 +1,131 @@
+function initGroupCable() {
+  var messages = $('#group-messages');
+  if(messages.length <= 0) return;
+
+  var groupId = messages.data('group-id');
+  var userId = messages.data('user-id');
+  var template = JST['templates/message'];
+
+  var page = 2;
+  var totalPages = 37;
+  var loadingMessages = false;
+
+  var prependMessage = function(message) {
+    messages.prepend(template({message: message, current_user: userId}));
+  };
+
+  var loadMessages = function() {
+    $.getJSON('/grupper/' + groupId + '/meddelanden/')
+      .then(function(resp) {
+        for (var message of resp.messages) prependMessage(message);
+        totalPages = resp.meta.total_pages;
+        to_bottom();
+      });
+  }();
+
+  var to_bottom = function() {
+    messages.scrollTop(messages.prop("scrollHeight"));
+  };
+
+  App.groups = App.cable.subscriptions.create({
+    channel: "GroupsChannel",
+    group_id: groupId },
+  { connected: function() {},
+    disconnected: function() {},
+    received: function(data) {
+      if (data['action'] == 'create') {
+        appendMessage(data['message'].message);
+      } else if (data['action'] == 'destroy') {
+        removeMessage(data['message_id']);
+      } else if (data['action'] == 'update') {
+        updateMessage(data['message'].message);
+      }
+    },
+    sendMessage: function(content) {
+      return this.perform('send_message', {
+        content: content,
+        group_id: groupId
+      });
+    },
+    destroyMessage: function(messageId) {
+      return this.perform('destroy_message', {
+        message_id: messageId
```

Comment:
Variables that come from the json data created in rails. Not going to edit that for some JS guidelines